hail/python/test/hail/utils/test_google_fs_utils.py

```diff
@@ -0,0 +1,100 @@
+import unittest
```

Comment:
This test doesn't need to fs backend specific.  I see now reason why wouldn't want to run using the Hadoop or gs backends.  The thing we need is to be able to create a globally visible tmp directory, wherever when using Hadoop, in gs when using the gs backend.  We have some existing utilities for this for Hadoop in the existing Scala code, but maybe cleanup needs to be thought through.

You don't need to change this now.